src/Tasks/FetchConversations.elm

```diff
@@ -15,15 +15,15 @@ type alias ConversationTask =
     Task (Result String ( Dict Int Conversation, Dict Int User )) (Result String ( Dict Int Conversation, Dict Int User ))
 
 
-fetchConversations : LoginInfo -> ConversationTask
-fetchConversations loginInfo =
+fetchConversations : LoginInfo -> Int -> ConversationTask
+fetchConversations loginInfo myUserId =
     { verb = "GET"
     , headers =
         [ ( "Authorization", "Basic:api:" ++ loginInfo.apiKey )
         , ( "Content-Type", "application/json;charset=UTF-8" )
         , ( "Accept", "application/json" )
         ]
-    , url = Http.url "https://app.communityshare.us/api/conversation" [ ( "user_id", "976" ) ]
+    , url = Http.url "https://app.communityshare.us/api/conversation" [ ( "user_id", toString myUserId ) ]
```

Comment:
this is a good catch. this code was in before I was loading the current user and that's why it was hard-coded. thanks for the catch. I'm sure you were surprised when you tried to run it :wink:

by the way, adding the extra `Int` is very normal

ironically, we don't exactly need it here, as we could do authentication with the `{ email, password }` from the logged-out dialog…